controllers/account.js

```diff
@@ -1,8 +1,81 @@
 const express = require('express');
 const router = express.Router();
+const db = require('../models');
+const logger = require('../logger');
+const bcrypt = require('bcrypt');
 
-router.put('/update-account', (req, res) => {
-    res.send('/update-account working');
+router.get('/get-users', (req, res) => {
+    db.user.findAll()
+    .then(data => {
+        res.status(200);
+        return res.json(data);
+    })
+    .catch(err => {
+        logger.error(err);
+        res.status(500);
+        return res.json({ errMessage: 'Server Error' });
+    });
+});
+
+router.get('/get-user-by-id/:id', (req, res) => {
```

Comment:
Great job!  Small suggestion, can this be renamed to `/get-user/:id`?  You used the style in chores and categories, so this will keep it consistent.

Reply:
You're right, I just fixed it.